src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -299,7 +309,8 @@ object ScalafixPlugin extends AutoPlugin {
         workingDirectory = (ThisBuild / baseDirectory).value.toPath,
         loadedRules = () => scalafixInterfaceProvider.value().availableRules(),
         terminalWidth = Some(JLineAccess.terminalWidth),
-        allowedTargetFilesPrefixes = Nil
+        allowedTargetFilesPrefixes = Nil,
+        jgitCompletion = (ThisBuild / scalafixJGitCompletion).value
```

Comment:
This shouldn't matter for most cases, but for the sake of potential extensibility we could let potential project-level overrides
```suggestion
        jgitCompletion = scalafixJGitCompletion.value
```